src/nmark/readInline.nim

```diff
@@ -57,29 +55,25 @@ proc readAutoLink*(line: string): seq[DelimStack] =
 
     of '<':
       if openNum == 0:      
-        resultSeq.add(DelimStack(position: i, typeDelim: "<", numDelim: 1, isActive: true, potential: canOpen))
+        result.add(DelimStack(position: i, typeDelim: "<", numDelim: 1, isActive: true, potential: canOpen))
```

Comment:
Here and below - just removed unnecessary `resultSeq`, it doesn't really have any effect on performance :)